### PR TITLE
Updated Enf 2 risk calculation blog post

### PR DESCRIPTION
**Pull request by HannaHei**

### CODE REVIEW

File: blog/2020-12-17-risk-calculation-enf-2-0/index_de.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- Gewichtete Kontaktzeit <15 Minuten: Begegnung mit niedrigem Risiko (grün)
+- Gewichtete Kontaktzeit >=15 Minuten: Begegnung mit erhöhtem Risiko (rot) 
+ 
+Abschließend werden alle Zeitfenster eines Tages betrachtet (die gewichteten Kontaktzeiten) summiert. Ist deren Summe >= 15 Minuten, wird der Gesamtstatus rot, ansonsten ist dieser grün.
+ 
+Letztlich bedeutet das: Mehrere „grüne“ Begegnungen (Zeitfenster) können in Summe zu einem roten Status führen.
+
+Wer noch tiefer in die Materie einsteigen möchte:
+
+- Es stehen die aktualisierten Diagramme auf Github: https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation
```
Comment (Ein-Tim) [inline blog/2020-12-17-risk-calculation-enf-2-0/index_de.md:50]:
```suggestion
- Es stehen aktualisierten Diagramme auf Github zur Verfügung: https://github.com/corona-warn-app/cwa-documentation/tree/master/images/risk_calculation
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
One german sentence does not make sense

File: blog/2020-12-17-risk-calculation-enf-2-0/index_de.md (hunk 1 above)
Comment (Ein-Tim) [reply blog/2020-12-17-risk-calculation-enf-2-0/index_de.md:50]: Resolved with: #700

**Comment by dsarkar** [comment]
Hi @HannaHei, this PR can be closed, correct? The blog is online already.

**Comment by HannaHei** [comment]
Done

**Pull request closed without merging**